common/lib/sandbox-packages/README

```diff
@@ -1 +0,0 @@
-This directory is in the Python path for sandboxed Python execution.
```

Comment:
It's great seeing all this code removed from edx-platform 😄 

I searched for `sandbox-packages` on your branch, and found two additional references still in edx-platform (not counting the README reference you already found). Please find those and clean them up too.